Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ValidatorData.xml

```diff
@@ -2950,6 +2956,18 @@
         <CombineArray value="AP_IsVanillaRoboticsFacility"/>
         <CombineArray value="AP_IsAPRoboticsFacility"/>
     </CValidatorCombine>
+    <CValidatorUnitType id="AP_IsAPRoboticsFacilityWarp">
+        <Value value="AP_RoboticsFacilityWarp"/>
+    </CValidatorUnitType>
+    <CValidatorUnitType id="AP_IsVanillaRoboticsFacilityWarp">
+        <Value value="RoboticsFacilityWarp"/>
+    </CValidatorUnitType>
+    <CValidatorCombine id="IsRoboticsFacilityWarp" parent="CValidatorCombine">
+        <!-- Used by Karax's Prodigal Engineer on Salvation -->
+        <!-- Override -->
+        <CombineArray value="AP_IsVanillaRoboticsFacilityWarp"/>
+        <CombineArray value="AP_IsAPRoboticsFacilityWarp"/>
+    </CValidatorCombine>

```

Comment:
Should be defined in ArchipelagoPlayerLotV as vanilla `StargateWarp` and `StargateWarp` aren't defined within scope of this mod